src/basic/path-util.h

```diff
@@ -9,99 +9,20 @@
 #include "string-util.h"
 #include "time-util.h"
 
-#define PATH_SPLIT_SBIN_BIN(x) x "sbin:" x "bin"
-#define PATH_NORMAL_SBIN_BIN(x) x "bin"
-
-#if HAVE_SPLIT_BIN
-#  define PATH_SBIN_BIN(x) PATH_SPLIT_SBIN_BIN(x)
-#else
-#  define PATH_SBIN_BIN(x) PATH_NORMAL_SBIN_BIN(x)
-#endif
-
-#define DEFAULT_PATH_NORMAL PATH_SBIN_BIN("/usr/local/") ":" PATH_SBIN_BIN("/usr/")
-#define DEFAULT_PATH_SPLIT_USR DEFAULT_PATH_NORMAL ":" PATH_SBIN_BIN("/")
-
-#if HAVE_SPLIT_USR
-#  define DEFAULT_PATH DEFAULT_PATH_SPLIT_USR
-#else
-#  define DEFAULT_PATH DEFAULT_PATH_NORMAL
-#endif
-
 bool is_path(const char *p) _pure_;
 bool path_is_absolute(const char *p) _pure_;
-int safe_getcwd(char **ret);
-int path_make_absolute_cwd(const char *p, char **ret);
 char* path_startswith(const char *path, const char *prefix) _pure_;
 int path_compare(const char *a, const char *b) _pure_;
 bool path_equal(const char *a, const char *b) _pure_;
 bool path_equal_or_files_same(const char *a, const char *b, int flags);
-char* path_join(const char *root, const char *path, const char *rest);
-char* path_simplify(char *path, bool kill_dots);
 
 /* Note: the search terminates on the first NULL item. */
```

Comment:
This comment belongs to PATH_IN_SET, so it should be removed as well.

Reply:
Done